sklearn/preprocessing/imputation.py

```diff
@@ -189,6 +194,13 @@ def _sparse_fit(self, X, strategy, missing_values, axis):
         else:
             X = X.tocsc()
 
+        if axis == 0:
+            mask = _get_mask(X.data, self.missing_values)
+            mask_matrix = X.__class__((mask, X.indices.copy(),
+                                       X.indptr.copy()), shape=X.shape,
+                                      dtype=X.dtype)
+            self.feat_with_missing_ = mask_matrix.sum(axis=0).A.nonzero()[1]
```

Comment:
You really shouldn't be using attributes like this to transfer information back to a calling function. That's what return values are for. Storing this as an attribute is redundant with `imputed_features_`